Projects/B-L072Z-LRWAN1/Applications/LoRa/AT_Slave/LoRaWAN/App/src/lora.c

```diff
@@ -439,59 +436,29 @@ void LORA_Init(LoRaMainCallback_t *callbacks, LoRaParam_t *LoRaParam)
   LoRaMacCallbacks.GetBatteryLevel = LoRaMainCallbacks->BoardGetBatteryLevel;
   LoRaMacCallbacks.GetTemperatureLevel = LoRaMainCallbacks->BoardGetTemperatureLevel;
   LoRaMacCallbacks.MacProcessNotify = LoRaMainCallbacks->MacProcessNotify;
-#if defined( REGION_AS923 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_AS923);
-  LoRaRegion = LORAMAC_REGION_AS923;
-#elif defined( REGION_AU915 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_AU915);
-  LoRaRegion = LORAMAC_REGION_AU915;
-#elif defined( REGION_CN470 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_CN470);
-  LoRaRegion = LORAMAC_REGION_CN470;
-#elif defined( REGION_CN779 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_CN779);
-  LoRaRegion = LORAMAC_REGION_CN779;
-#elif defined( REGION_EU433 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_EU433);
-  LoRaRegion = LORAMAC_REGION_EU433;
-#elif defined( REGION_IN865 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_IN865);
-  LoRaRegion = LORAMAC_REGION_IN865;
-#elif defined( REGION_EU868 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_EU868);
-  LoRaRegion = LORAMAC_REGION_EU868;
-#elif defined( REGION_KR920 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_KR920);
-  LoRaRegion = LORAMAC_REGION_KR920;
-#elif defined( REGION_US915 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_US915);
-  LoRaRegion = LORAMAC_REGION_US915;
-#elif defined( REGION_RU864 )
-  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, LORAMAC_REGION_RU864);
-  LoRaRegion = LORAMAC_REGION_RU864;
-#else
-#error "Please define a region in the compiler options."
-#endif
+
+  LoRaMacInitialization(&LoRaMacPrimitives, &LoRaMacCallbacks, region);
+  LoRaRegion = region;
 
 #if defined( HYBRID )
```

Comment:
Not sure if HYBRID can be always defined or if we should introduce two more "fake bands"  (`LORAMAC_REGION_US915_HYBRID` and `LORAMAC_REGION_AU915_HYBRID`)